$NetBSD: patch-configure,v 1.2 2015/10/03 12:08:37 bsiegert Exp $

Don't add -ldl for all OS; this gets auto-detected somewhere else.

--- configure.orig	2015-10-02 05:19:31.000000000 +0200
+++ configure	2015-10-02 05:19:34.000000000 +0200
@@ -17321,8 +17321,6 @@
 case "$host_os" in
 *mingw* | *cygwin*)
     LDFLAGS="$LDFLAGS -no-undefined -lpsapi -lwldap32 -lws2_32" ;;
-*)
-    LDFLAGS="$LDFLAGS -ldl" ;;
 esac
 
 # ---------------------------------------------------------------------------
